sound/soc/sof/intel/hda-dai-ops.c

```diff
@@ -277,12 +277,37 @@ static const struct hda_dai_widget_dma_ops hda_ipc4_dma_ops = {
 	.post_trigger = hda_ipc4_post_trigger
 };
 
+static int hda_ipc3_post_trigger(struct snd_sof_dev *sdev, struct snd_soc_dai *cpu_dai,
+				 struct snd_pcm_substream *substream, int cmd)
```

Comment:
the commit message describes two independent parts, could the IPC4 fix for suspend while paused be handled with a dedicated commit?

Reply:
Unfortunately no I cant split that part. The use of post_trigger() in suspend is an automatic fix for IPC4 that we get for free.

Reply:
ok